.github/workflows/policy_controller.yml

```diff
@@ -190,31 +201,27 @@ jobs:
           docker pull ghcr.io/linkerd/proxy-init:$PROXY_INIT_VERSION
           docker image ls | grep ghcr.io/linkerd
           tag="$(CI_FORCE_CLEAN=1 bin/root-tag)"
-          k3d image import \
-            ghcr.io/linkerd/controller:$tag \
-            ghcr.io/linkerd/policy-controller:$tag \
-            ghcr.io/linkerd/proxy:$tag \
-            ghcr.io/linkerd/proxy-init:$PROXY_INIT_VERSION
+          # Image loading is flakey in CI, so retry!
+          until k3d image import \
```

Comment:
is it worth limiting this to a fixed number of retries?  having something wrong here but needing to wait the full 20m timeout before finding out seems like it could potentially be a pain point

Reply:
Yeah, I went back and forth on that. I can't really think of scenarios where this could fail for a legitimate reason short of the CI workflow itself being broken...